.github/workflows/test.yaml

```diff
@@ -62,10 +73,12 @@ jobs:
           $env:CMAKE_SYSTEM_VERSION="10.0.22621.0"
           $env:PATH="$gopath;$gccpath;$env:PATH"
           echo $env:PATH
-          go generate -x ./...
+
+          $env:GOARCH = ""; go run build.go -f -d -target=${{ matrix.arch }}
```

Comment:
Instead of unsetting GOARCH in each steps, maybe remove the env var on line 60?

Reply:
After further thought, I think the clearing of the env should stay our of shear paranoia that it trips up someone later if it is removed now.

Reply:
I'd also argue this way also clearly shows that `go run` must be allowed to do its default thing.